### PR TITLE
(GH-77) Set TEMP folder to include vagrant vmId

### LINKED ISSUE #77: Set TEMP path locally before running vagrant - note for locked files
Related to #50, we need to isolate the TEMP environments so that each instance doesn't step on the other. That is to set a TEMP area that we can have each vagrant run in when running the service.

Keep in mind there is something we look for to fail the environment and that is : https://github.com/chocolatey/package-verifier/commit/5b02947249bb2f5dd45ca1b78d5fc42d2bc7d9bb#diff-943f03b4181625724f13ae8ef798df44R67. This could be a local file to the host machine and not in the machine like we may have originally thought. It's best to isolate these environments entirely.

~~~
2019-09-11 06:08:38,827 14 [WARN ] -  [Vagrant][Error] The following WinRM command responded with a non-zero exit status.
2019-09-11 06:08:38,827 14 [WARN ] -  [Vagrant][Error] Vagrant assumes that this means the command failed!
2019-09-11 06:08:38,828 14 [WARN ] -  [Vagrant][Error] powershell -ExecutionPolicy Bypass -OutputFormat Text -file c:\tmp\vagrant-shell.ps1
2019-09-11 06:08:38,828 14 [WARN ] -  [Vagrant][Error] Stdout from the command:
2019-09-11 06:08:38,828 14 [WARN ] -  [Vagrant][Error] Stderr from the command:
2019-09-11 06:08:38,828 14 [WARN ] -  [Vagrant][Error] del : Cannot remove item C:\Windows\Temp\WinRM_Elevated_Shell.log: The process 
2019-09-11 06:08:38,828 14 [WARN ] -  [Vagrant][Error] cannot access the file 'C:\Windows\Temp\WinRM_Elevated_Shell.log' because it 
2019-09-11 06:08:38,829 14 [WARN ] -  [Vagrant][Error] is being used by another process.
2019-09-11 06:08:38,829 14 [WARN ] -  [Vagrant][Error] At C:\tmp\vagrant-elevated-shell.ps1:19 char:3
2019-09-11 06:08:38,829 14 [WARN ] -  [Vagrant][Error] +   del $out_file
2019-09-11 06:08:38,829 14 [WARN ] -  [Vagrant][Error] +   ~~~~~~~~~~~~~
2019-09-11 06:08:38,830 14 [WARN ] -  [Vagrant][Error]     + CategoryInfo          : WriteError: (C:\Windows\Temp\WinRM_Elevated_Shel 
2019-09-11 06:08:38,830 14 [WARN ] -  [Vagrant][Error]    l.log:FileInfo) [Remove-Item], IOException
2019-09-11 06:08:38,830 14 [WARN ] -  [Vagrant][Error]     + FullyQualifiedErrorId : RemoveFileSystemItemIOError,Microsoft.PowerShell 
2019-09-11 06:08:38,830 14 [WARN ] -  [Vagrant][Error]    .Commands.RemoveItemCommand
~~~

### PR DESCRIPTION
**Pull request by mkevenaar**
This will make sure the temp folder for the vagrant actions is set
to a known empty folder, preventing lock issues.

Fixes #77 

/cc @ferventcoder @gep13

### CODE REVIEW

File: src/chocolatey.package.verifier/infrastructure.app/tasks/TestPackageTask.cs [hunk 1]
```diff
@@ -370,15 +376,17 @@ private void test_package(VerifyPackageMessage message)
...
             {
                 Bootstrap.handle_exception(ex);
             }
-            //finally
-            //{
-            //    TransactionLock.release(VAGRANT_LOCK_NAME, lockTaken: true);
-            //}
+            finally
+            {
+                Environment.SetEnvironmentVariable("TEMP", "C:\\Windows\\Temp");
```
Comment (ferventcoder) [inline src/chocolatey.package.verifier/infrastructure.app/tasks/TestPackageTask.cs]: It's a process, don't set it back.

**Comment by ferventcoder** [comment]
This looks pretty good. I have that one thing I'd like to see.

File: src/chocolatey.package.verifier/infrastructure.app/tasks/TestPackageTask.cs (hunk 1 above)
Comment (mkevenaar) [reply src/chocolatey.package.verifier/infrastructure.app/tasks/TestPackageTask.cs]: Changed.

**Comment by mkevenaar** [comment]
@ferventcoder I have made the changes as requested.

**Pull request merged**